docs/reference/dockerd.md

```diff
@@ -892,7 +892,7 @@ Alternatively, you can set custom locations for CDI specifications using the
 When CDI is enabled for a daemon, you can view the configured CDI specification
 directories using the `docker info` command.
 
-#### <a name="log-format"></a> Daemon logging format
+#### Daemon logging format {#log-format}
```

Comment:
I think we use this approach for all of the CLI reference docs (and use that format to match sections with the options table); is it an issue for those as well?

Reply:
Ah! That way, gotcha